### PR TITLE
main/btrfs-progs: upgrade to 4.16.1

### PR DESCRIPTION
**Pull request by prspkt**

### CODE REVIEW

File: main/btrfs-progs/APKBUILD [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
+makedepends="$depends_dev acl-dev asciidoc attr-dev autoconf automake e2fsprogs-dev
 	libtool lzo-dev py3-setuptools python3-dev util-linux-dev xmlto zlib-dev"
 subpackages="$pkgname-doc $pkgname-dev $pkgname-bash-completion:bashcomp:noarch
 	py-$pkgname:py $pkgname-libs $pkgname-extra"
 source="https://www.kernel.org/pub/linux/kernel/people/kdave/$pkgname/$pkgname-v$pkgver.tar.xz
 	includes.patch
 	musl-compat.patch
 	"
-builddir="$srcdir/$pkgname-v$pkgver"
+builddir="$srcdir"/$pkgname-v$pkgver
```
Comment (jirutka) [inline main/btrfs-progs/APKBUILD]: Why this change? The current form is preferred.

Comment (prspkt) [reply main/btrfs-progs/APKBUILD]: @jirutka Noted. Will amend in future commits.
